src/app/material/TranslatedMatPaginator.ts

```diff
@@ -0,0 +1,30 @@
+import { MatPaginatorIntl } from '@angular/material/paginator';
+import { TranslateService } from '@ngx-translate/core';
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class TranslatedMatPaginator extends MatPaginatorIntl {
+  constructor(private translateService: TranslateService) {
+    super();
+  }
+
+  getRangeLabel = (page: number, pageSize: number, length: number) => {
+    this.getAndInitTranslations();
+    length === 0 || pageSize === 0 ? (page = 0) : (page = page + 1);
```

Comment:
```suggestion
    const newPage = (length === 0 || pageSize === 0) ? 0 : page + 1;
```
tránh modify biến đã có hoặc đặt tên biến mới trùng biến cũ, e có thể tạo thêm biến nếu cần